dg-common/src/main/java/org/finra/datagenerator/common/SocialNetwork_Example_Java/SocialNetworkUtilities.java

```diff
@@ -29,16 +29,30 @@
  * stub-to-data translation.
  */
 public class SocialNetworkUtilities {
+    /**
+     * Randomly get whether it is a secret account
+     * @return Boolean
+     */
     public static Boolean getRandomIsSecret() {
         return RandomHelper.evaluateProbability(0.1); // 10% of accounts are secret accounts.
     }
 
+    /**
+     * Get random geographical location
+     * @return 2-Tuple of ints (latitude, longitude)
+     */
     public static Tuple2<Double,Double> getRandomGeographicalLocation() {
         return new Tuple2<>(
             (double)(RandomHelper.randWithConfiguredSeed().nextInt(999) + 1) / 100,
             (double)(RandomHelper.randWithConfiguredSeed().nextInt(999) + 1) / 100);
     }
 
+    /**
+     * Get distance between geographical coordinates
+     * @param point1 Point1
+     * @param point2 Point2
+     * @return Distance (double)
+     */
     public static Double getDistanceBetweenCoordinates(Tuple2<Double,Double> point1, Tuple2<Double,Double> point2) {
         // sqrt( (x2-x1)^2 + (y2-y2)^2 )
```

Comment:
This is not commented code -- it is an explanation of what the lines below are doing.